labs/lab-03/tasks/goto-algs/binary_search/support/Makefile

```
@@ -1,8 +1,8 @@
 CC = gcc
-CFLAGS = -Wall -g
+CFLAGS = -Wall
```

```suggestion
CFLAGS = -Wall -g
```
Why remove `-g`? Students don't use GDB often so that `-g` is required, but it doesn't do any harm either.